Automattic/wp-calypso
# Features list: Fix text under /start/plans page

**Proposed Changes**

Change text under `/start/plans` in order to reflect what exists
under `/pricing` page.


For reference, here is the page under `/pricing`, and the change we need to change under `/start/plans` page:

![Screenshot_2022-09-20_at_12 57 38_PM](https://user-images.githubusercontent.com/1182160/191930610-16c4e325-e5ab-4762-bb52-249334317a04.png)

Done is:

- [x] The copy is consistent with the logged-out views.
- [x] The new features are listed on the /plans page during singup.

#### Testing Instructions

How to test:
- Visit `wordpress.com/start/plans`. Feature text under `Business` and `eCommerse` plans is: `SFTP (SSH File Transfer Protocol) and Database Access`. This step is about about checking **current behavior that is not desired**
-  Visit `/start/plans` locally. Please mention that we should select a domain after navigating to this page. Feature text under `Business` and `eCommerse` plans is: `SFTP, SSH, WP-CLI, and Database access`

In order to provide as much help as possible here are some _before_ and _after_ screenshots:

`/start/plans` was _before_ the fix:

<img width="1338" alt="Screenshot 2022-09-23 at 11 28 40" src="https://user-images.githubusercontent.com/1182160/191931728-e0c268df-9dee-438e-962a-6cc79cbd3bd8.png">

And _after_ the fix:

![after](https://user-images.githubusercontent.com/497103/193849440-6eb0d6c7-5ca0-4de1-8ee1-52c500ffcb52.png)

#### Pre-merge Checklist






Related to #68193
